### DISCUSSION TITLE
"Istio sets the sidecar CRD for the current namespace, but services in other namespaces are still accessible.

### DISCUSSION THREAD

**Discussion by 18515291426** [istio/istio#53084] [Q&A]
"Istio sets the sidecar CRD for the current namespace, but services in other namespaces are still accessible. I hope someone can help answer this question. Thank you!
![9f8126ef333eda9c0230479f87a2459](https://github.com/user-attachments/assets/b0ade37b-fc4e-43de-9db1-173c70a2b810)

**Comment by howardjohn** [comment]
see https://istio.io/latest/docs/ops/configuration/traffic-management/traffic-routing/#unmatched-traffic. Sidecar doesn't "block" traffic it just controls what config the proxy yas

**Reply by 18515291426** [reply]
In my screenshot example, the outbound traffic is set to the current namespace, but services in other namespaces are still accessible. Is this result correct? I found this in the official documentation: https://istio.io/latest/docs/reference/config/networking/sidecar/, which mentions that traffic can be restricted

**Reply by 18515291426** [reply]
![image](https://github.com/user-attachments/assets/0fb92fdf-bd2f-4d9b-a91e-2d537d77dc64)

**Reply by howardjohn** [reply]
that doc is misleading, needs an update
